extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -609,10 +609,11 @@ angular.module('PaperUI.controllers.configuration', [ 'PaperUI.constants' ]).con
             dict.bridgeUID = thing.bridgeUID
             update = true;
         }
-        if (update) {
+        originalThing.configuration = thing.configuration = {};
+        if (JSON.stringify(originalThing) !== JSON.stringify(thing)) {
             thingService.update({
                 thingUID : thing.UID
-            }, dict);
+            }, thing);
```

Comment:
afaik, we do separate calls for configuration updates and thing updates, so I assume that this is fully by intention here.
But @aounhaider1 can probably give a better answer ;-)